Userland/Libraries/LibSQL/Heap.cpp

```diff
@@ -143,32 +186,34 @@ void Heap::flush()
             VERIFY_NOT_REACHED();
         }
         dbgln_if(SQL_DEBUG, "Flushing block {} to {}", block, name());
-        write_block(block, buffer_or_empty.value());
+        TRY(write_block(block, buffer_or_empty.value()));
     }
     m_write_ahead_log.clear();
     dbgln_if(SQL_DEBUG, "WAL flushed. Heap size = {}", size());
+    return {};
 }
 
-constexpr static const char* FILE_ID = "SerenitySQL ";
+#define SERENITY_SQL "SerenitySQL"
+constexpr static const char* FILE_ID = SERENITY_SQL " ";
```

Comment:
Hmm I would expect these to be constexpr StringViews, at least to avoid the `strlen` calls below at runtime
(I know `FILE_ID` already exists, I didn't catch this before, sorry).